src/App.css

```diff
@@ -1,38 +1,35 @@
-.App {
-  text-align: center;
+#root {
+  width: 700px;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+input {
+  background: gray;
+  color: white;
+  font-weight: bold;
+  grid-column-start: 1;
+  grid-column-end: 5;
+  padding: 10px;
+  height: 100px;
+  width: 676px;
+  text-align: right;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+button {
```

Comment:
- buttons must take up 25% of the container width except the `0` button.
- button text should be black and centered
- make `wide` property a boolean. `true` means the `0` button takes up `50%` of space.

Reply:
button is taking 1/4th = 25% width